src/Builder/SymbolTable.hpp

```diff
@@ -47,30 +47,33 @@ struct VariableScope {
    */
   bool contain(const std::string &name) const;
 
+  const std::string &getIdentifier() const { return identifier; }
+
+private:
   /*!
    * Identifier of the current scope, used for debugging and sanity check.
    */
   const std::string identifier;
 
-private:
   /*!
    * A mapping between symbol name and symbol value.
    */
-  std::map<std::string, T> _nameToValue;
+  std::map<std::string, T> nameToValue;
```

Comment:
Can you clarify on the proper usage of "_", I see that you added in some places, and removed in others.

Reply:
got it. I personally use the same name as it works (constructor parameter and class/struct variable) but I can see that using a "_" may be clearer.